aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js

```diff
@@ -79,10 +54,13 @@ const StepOne = () => {
               label="Email"
               htmlFor="twocolumn-textinput"
               name="twocolumn-textinput"
-              error={error.email}
-              onChange={() =>
-                attemptedAdvance && setError(stepOneValidate(formValues))
-              }
+              validate={{
```

Comment:
I think we only want this to validate onBlur/onChange if the user has already tried to advance to the next step. My understanding was we don't want to run validation as they're first trying to go through inputs because popping errors up prematurely can feel alarming. 

What I noticed here was if I typed an invalid email and pressed "tab" to move on to the next input, the validation ran and the email input got put in focus right away. Expected would be if this is my first attempt at entering info, no validation until I press "Next".

Reply:
^^ Was testing on `WizardValidationExample` but same behavior should apply here.

Reply:
Thanks Taylor. I was wondering about this. The main reason I did onBlur is it was easiest to make the error message at the bottom come and go at the right time. It's confusing if it sticks around until Next is hit again. I think the other way around this would to put back in the 'attemptedAdvance' flag and something in the onChange handlers although it seemed very awkward to implement. Thoughts on a better way to provide that message at the bottom?

Reply:
Fixed as noted below.